corehq/apps/users/models.py

```diff
@@ -3003,6 +3003,36 @@ class UserOptionToggles(models.Model):
     use_latest_build_cloudcare = models.BooleanField(default=False)
 
 
+def is_option_enabled(option, username=None, domain_name=None):
+    """
+    Returns whether an option is enabled for a user or their domain.
+
+    Returns False if the option does not exist.
+    """
+    assert username or domain_name, 'username and/or domain_name required'
+
+    if username:
+        couch_user = CouchUser.get_by_username(username)
+        assert couch_user, f'User {username} not found'
+        django_user = couch_user.get_django_user()
+        try:
+            options = django_user.option_toggles
+            return getattr(options, option, False)
```

Comment:
what's this `False` doing there?  Is this for the possibility of toggles being configured only at the domain level?

Reply:
Exactly. This is to allow for a domain-only toggle.